_docs/4-kubernetes/1-controlplane.md

```diff
@@ -17,37 +17,52 @@ order: 2
 
 ### Deploying Amalgam8 Control Plane <a id="deploy"></a>
 
-Amalgam8 rus a control-loop to validate routing rules stored in Third Party Resources (TPR)
+Amalgam8 runs a control-loop to validate routing rules stored in Third Party Resources (TPR)
```

Comment:
This is inaccurate. The controller validates incoming rules before storing them as TPRs.